### PR TITLE
merge v3.0.1 into v3.0.x

### PR DESCRIPTION
**Pull request by psotofer**
Please, review pull into v3.0.x

### CODE REVIEW

**Review by arillpa — COMMENTED** [review]
checked and approved

**Review by arillpa — COMMENTED** [review]
Fix url in technical code lists - checked and approved

**Review by arillpa — COMMENTED** [review]
Fix italics format in code list name - checked and validated

**Review by arillpa — COMMENTED** [review]
https://github.com/OP-TED/espd-docs/pull/1/commits/1ef4df9fcc06bece2fdbc537b4e1ebfaf05ed921
6.8_Evidences.adoc

Please replace 'modeel' with 'model'

cac:PartyIdentification checked and validated``

**Review by arillpa — COMMENTED** [review]
https://github.com/OP-TED/espd-docs/pull/1/commits/322c806ace59c273d150fac1088bfc62e4180df9
update code list agency and version id:

- version id 3.0.1 checked and validated
- code list AgencyID 'EU-COM-GROW' still wrong, it should be 'OP'. To be changed for v3.1.0 in the whole documentation and xml examples.
- code list Agency in TenderingCriterionResponse 'ES-AEAT' not sure where it comes from

File: modules/ROOT/pages/Tech_AnnexI_Code_Lists_Catalogue.adoc [hunk 1]
```diff
@@ -120,7 +120,7 @@ An example of how to use this code list can be found in xref:Tech_AnnexII_XML_Ex
```
Comment (arillpa) [inline modules/ROOT/pages/Tech_AnnexI_Code_Lists_Catalogue.adoc:117]: this is a test ignore this. The action is already validated

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -8,5 +8,5 @@ include::partial$attributes.adoc[]
 
 * xref:xml_business_handbook.adoc[ESPD BUSINESS HANDBOOK]
 * xref:xml_technical_handbook.adoc[ESPD TECHNICAL HANDBOOK]
-* link:{attachmentsdir}/ESPD-data-model-html/index.html[ESPD-EDM DATA MODEL]
+* link:{attachmentsdir}/ESPD-data-model-html/index.html[ESPD-EDM DATA MODEL HTML]
```
Comment (psotofer) [inline modules/ROOT/pages/index.adoc:11]: Validated.

File: modules/ROOT/attachments/ESPD-data-model-html/EARoot/EA1.htm [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (arillpa) [inline modules/ROOT/attachments/ESPD-data-model-html/EARoot/EA1.htm:1]: Update html data model

date formats validated

File: modules/ROOT/pages/6.2_ESPD_Response_XSD_Schema.adoc [hunk 4]
```diff
@@ -1,7 +1,7 @@
 = ESPD Response XSD Schema
 include::partial$attributes.adoc[]
 
-For the ESPD Response, the ESPD-EDM V3.0.0 uses the UBL-2.3 document named QualificationApplicationResponse.xsd XSD Schema. This schema can be found under the folder link:{url-tree-tmp}/dist/xsdrt/maindoc[dist/xsdrt/maindoc] (or the equivalent documented xsd folder).
+For the ESPD Response, the ESPD-EDM V3.0.0 uses the UBL-2.3 document named QualificationApplicationResponse.xsd XSD Schema. This schema can be found under the folder link:{url-tree-tmp}/ubl-2.3/xsdrt/maindoc[ubl-2.3/xsdrt/maindoc] (or the equivalent documented xsd folder).
```
Comment (arillpa) [inline modules/ROOT/pages/6.2_ESPD_Response_XSD_Schema.adoc:4]: ubl-2.3/xsdrt/maindoc
validated

File: modules/ROOT/pages/6.3_ESPD_Response_cardinalities.adoc [hunk 5]
```diff
@@ -15,7 +15,7 @@ If you compare both figures you will observe that:
 . The cardinalities of the root common basic components, such as `cbc:ID`, `cbc:UUID`, `cbc:TendererRole`, and other are different for the ESPD than for UBL-2.3;
 . Similarly, the cardinalities of aggregate components like `cac:ContractingParty`, `cac:ProcurementProjectLot` and `cac:TenderingCriterion` are different to the UBL-2.3 ones.
 
-The cardinality constraints added by the ESPD are not defined in the XSD Schema. In order to control these constraints the ESPD-EDM uses link:https://schematron.com/[ISO Schematron] assertions. The ESPD-EDM V3.0.0 distribution package provides Schematron schemata and CVA files for the validation of the XML instances (folder link:{url-tree-tmp}/dist/[_dist/val_]).
+The cardinality constraints added by the ESPD are not defined in the XSD Schema. In order to control these constraints the ESPD-EDM uses link:https://schematron.com/[ISO Schematron] assertions. The ESPD-EDM V3.0.0 distribution package provides Schematron schemata and CVA files for the validation of the XML instances (folder link:{url-tree-tmp}/validation[_/validation_]).
```
Comment (arillpa) [inline modules/ROOT/pages/6.3_ESPD_Response_cardinalities.adoc:18]: folder link:{url-tree-tmp}/validation[_/validation_
validated

File: modules/ROOT/pages/6.5_Reference_to_publications_and_to_ESPD_Request.adoc [hunk 6]
```diff
@@ -66,14 +66,14 @@ _Table 27. Reference to the ESPD Request, expected elements_
 |*cbc:DocumentTypeCode* |Code |1 |The type of document being referenced, expressed as a code. a|
 *Information Requirement*: link:http://wiki.ds.unipi.gr/display/ESPDInt/BIS+41+-+ESPD+V2.1.0#BIS41-ESPDV2.1-tbr92-013[_tbr92-013_].
 
-*Rule*: For the ESDP-EDM it is compulsory use of the value ESPD_REQUEST from the Code List link:{url-tree-tmp}/dist/cl/ods/[_doc-ref-content-type_]. See also the XML example below on how to specify the ESPD Request.
+*Rule*: For the ESDP-EDM it is compulsory use of the value ESPD_REQUEST from the Code List link:{url-tree-tmp}/codelists/gc/docref-content-type.gc[_doc-ref-content-type_]. See also the XML example below on how to specify the ESPD Request.
```
Comment (arillpa) [inline modules/ROOT/pages/6.5_Reference_to_publications_and_to_ESPD_Request.adoc:69]: codelists/gc/docref-content-type.gc
link does not work

File: modules/ROOT/pages/5.5_Financial_ratios.adoc [hunk 7]
```diff
@@ -36,7 +36,7 @@ _Figure 135. 'Financial ratio' EO mock-up_
 REQUIREMENT(s) specified by the buyers can be place outside a group of QUESTION(s) (see any other previous criteria) or *inside* a group of *QUESTION(s)*, which is the case for financial ratios, as you can see in the data structure for this criterion, below.
 ====
 
-Notice how the spreadsheet has been used to specify the three different financial ratios of the above mock-up example: the XML example below was produced using a XSL-T transformation link:{url-blob}/dist/xslt/[style-sheet] that takes the link:{url-blob}/dist/cl/ods/[ODS spreadsheet] and produced the link:{url-blob}/dist/xml/[XML instance].
+Notice how the spreadsheet has been used to specify the three different financial ratios of the above mock-up example: the XML example below was produced using a XSL-T transformation link:{url-tree-tmp}/xml-examples/__xslt__/ods-data-structures-to-ESPD-xml/ESPDRequest-Annotated.xslt[ESPDRequest-Annotated] that takes the link:{url-tree-tmp}/xml-examples/ESPD-criterion.xml[ESPD-criterion] and produced the link:{url-tree-tmp}/xml-examples/ESPD-Request.xml[ESPD-Request].
```
Comment (arillpa) [inline modules/ROOT/pages/5.5_Financial_ratios.adoc:39]: ESPDRequest-Annotated link does not work

File: modules/ROOT/pages/6.5_Reference_to_publications_and_to_ESPD_Request.adoc [hunk 8]
```diff
@@ -66,14 +66,14 @@ _Table 27. Reference to the ESPD Request, expected elements_
 |*cbc:DocumentTypeCode* |Code |1 |The type of document being referenced, expressed as a code. a|
 *Information Requirement*: link:http://wiki.ds.unipi.gr/display/ESPDInt/BIS+41+-+ESPD+V2.1.0#BIS41-ESPDV2.1-tbr92-013[_tbr92-013_].
 
-*Rule*: For the ESDP-EDM it is compulsory use of the value ESPD_REQUEST from the Code List link:{url-tree}/dist/cl/ods/[_doc-ref-content-type_]. See also the XML example below on how to specify the ESPD Request.
+*Rule*: For the ESDP-EDM it is compulsory use of the value ESPD_REQUEST from the Code List link:{url-tree-tmp}/dist/cl/ods/[_doc-ref-content-type_]. See also the XML example below on how to specify the ESPD Request.
```
Comment (arillpa) [inline modules/ROOT/pages/6.5_Reference_to_publications_and_to_ESPD_Request.adoc]: url-tree-tmp}/dist/cl/ods/[_doc-ref-content-type_ link does not work

File: modules/ROOT/pages/6.6_Economic_Operator.adoc [hunk 9]
```diff
@@ -324,7 +324,7 @@ _Table 36. Economic operator postal address, expected elements_
 |*cac:Country/cbc:IdentificationCode* |Code |1 |A code that identifies the country. a|
 *Information Requirement*: link:http://wiki.ds.unipi.gr/display/ESPDInt/BIS+41+-+ESPD+V2.1.0#BIS41-ESPDV2.1-tbr92-012[_tbr92-012_].
 
-*Rule*: The country of the contracting body must always be specified. Compulsory use of the Code List link:{url-tree}/dist/cl/ods/[Country from EU Vocabularies, which is based on ISO.]  
+*Rule*: The country of the contracting body must always be specified. Compulsory use of the Code List link:{url-tree-tmp}/dist/cl/ods/[Country from EU Vocabularies, which is based on ISO.]
```
Comment (arillpa) [inline modules/ROOT/pages/6.6_Economic_Operator.adoc]: Country from EU Vocabularies, which is based on ISO.
I think this text should not be part of the link

File: modules/ROOT/pages/6.6_Economic_Operator.adoc [hunk 10]
```diff
@@ -411,7 +411,7 @@ _Table 38. Service provider, expected elements_
 |*cac:PostalAddress/cac:Country/cbc:IdentificationCode* |Code |1 |The code that identifies the country of the service provider. a|
 *Information Requirement*: link:http://wiki.ds.unipi.gr/display/ESPDInt/BIS+41+-+ESPD+V2.1.0#BIS41-ESPDV2.1-tbr070-021[_tbr070-021_].
 
-*Rule*: The country of the service provider must always be specified. Compulsory use of the Code List link:{url-tree}/dist/cl/ods/[Country from EU Vocabularies, which is based in ISO].
+*Rule*: The country of the service provider must always be specified. Compulsory use of the Code List link:{url-tree-tmp}/dist/cl/ods/[Country from EU Vocabularies, which is based in ISO].
```
Comment (arillpa) [inline modules/ROOT/pages/6.6_Economic_Operator.adoc]: Country from EU Vocabularies, which is based in ISO
I think this text should not be part of the link, only the word country

File: modules/ROOT/pages/1.5_Codes_and_Identifiers.adoc [hunk 11]
```diff
@@ -146,14 +146,14 @@ This fragment of XML shows how the compulsory attributes are used for the some o
 . The UUID follows the ISO/IEC Scheme 9834-8:2008 Version 4 and was generated by the European Commission's Directorate General GROWTH (DG GROW)
 . The reference number used to identify to which procurement procedure this ESPD document belongs (PP.20170419.1024-9) has been supplied by the Spanish Directorate DGPE
 . Generic information, such as the content version ID, use always by default the "EU-COM-GROW" Agnecy ID. Notice that the other additional attributes may be also used, as in this example.
-. Beware that the codes may be numbers, text or combinations of both. These code labels are the ones that are specified in the codelist spreadsheets and XML Genericode files distributed jointly with this specification (in the folder /codelists of the link:{url-tree}/dist[distribution package].
+. Beware that the codes may be numbers, text or combinations of both. These code labels are the ones that are specified in the codelist spreadsheets and XML Genericode files distributed jointly with this specification (in the folder /codelists of the link:{url-tree-tmp}/dist[distribution package].
```
Comment (arillpa) [inline modules/ROOT/pages/1.5_Codes_and_Identifiers.adoc:149]: point 6, link:{url-tree-tmp}/dist[distribution package does not work

File: modules/ROOT/pages/3.4_Properties.adoc [hunk 12]
```diff
@@ -56,7 +56,7 @@ Caption (i.e. a 'label'), specific MS or buyer requirement (e.g. 'Number of refe
 |*cbc:TypeCode* |Code |1 |The type of property. Used to verify that structure of the property is correct. a|
 *Information Requirement*: link:http://wiki.ds.unipi.gr/display/ESPDInt/BIS+41+-+ESPD+V2.1.0#BIS41-ESPDV2.1.0-tbr070-013[_tbr070-013_].
 
-*Rule*: Compulsory use of the link:{url-tree}/codelists/gc/CriterionElementType.gc[Code List 'CriterionElementType']. Possible types are 'CAPTION, REQUIREMENT and QUESTION'. If the type is CAPTION or REQUIREMENT no answer is expected from the economic operator and therefore the cbc:ValueDataTypeCode must be set to NONE. Otherwise this value must be set to one of the values defined in the link:{url-tree}/codelists/gc/ResponseDataType.gc[Code List 'ResponseDataType']
+*Rule*: Compulsory use of the link:{url-tree-tmp}/codelists/gc/CriterionElementType.gc[Code List 'CriterionElementType']. Possible types are 'CAPTION, REQUIREMENT and QUESTION'. If the type is CAPTION or REQUIREMENT no answer is expected from the economic operator and therefore the cbc:ValueDataTypeCode must be set to NONE. Otherwise this value must be set to one of the values defined in the link:{url-tree-tmp}/codelists/gc/ResponseDataType.gc[Code List 'ResponseDataType']
```
Comment (arillpa) [inline modules/ROOT/pages/3.4_Properties.adoc:59]: Code List 'CriterionElementType', Code list should not be part of the link

File: modules/ROOT/pages/3.7_XML_examples_and_tools.adoc [hunk 13]
```diff
@@ -3,7 +3,7 @@ include::partial$attributes.adoc[]
 
 The fact of presenting the data structures as a spread-sheet book had an additional reason: *to use the spread-sheet as an elementary prototype tool to generate the XML instances of the criteria for the ESPD Request and ESDP Response documents*.
 
-Thus the folder link:{url-tree}/xml-examples/__xslt__/ods-data-structures-to-ESPD-xml[xml-examples/__xslt__/ods-data-structures-to-ESPD-xml] contains four XSL style-sheets that facilitate the generation of the complete set of criteria required in an ESDP Request or in an ESDP Response XML file.
+Thus the folder link:{url-tree-tmp}/xml-examples/__xslt__/ods-data-structures-to-ESPD-xml[xml-examples/__xslt__/ods-data-structures-to-ESPD-xml] contains four XSL style-sheets that facilitate the generation of the complete set of criteria required in an ESDP Request or in an ESDP Response XML file.
```
Comment (arillpa) [inline modules/ROOT/pages/3.7_XML_examples_and_tools.adoc:6]: link:{url-tree-tmp}/xml-examples/__xslt__/ods-data-structures-to-ESPD-xml[xml-examples/__xslt__/ods-data-structures-to-ESPD-xml] does not work

File: modules/ROOT/pages/2.6_EU_and_national_publications.adoc [hunk 14]
```diff
@@ -193,10 +193,10 @@ The XML snippet below illustrates how to use the cac:AdditionalDocumentReference
...
 . In the case of publication on the TED, at ESPD Request preparation time, the procurer does not have yet the OJS number, but can receive a temporary CN identifier (Received Notice Number). To save this number in the DocumentReference component create two cbc:Description elements and use the second one to place this Received Notice Number. If you do not have a description for the CN then use a 'dummy' content for the first description, e.g. <cbc:Description>_</cbc:/Description>.
 . Notice how the schemeAgencyID is used in this example to specify that the national Agency that created this reference to the national publication (e.g. "PLACE" to refer to the ID of the national electronic board where the CN was published). The pattern of the ID is the one created by the Spanish buyer.
-. In this case the code NOJCN means 'Contract Notice (CN) published on a National Government Official Journal' (see file ESPD-CodeLists-V3.0.0) in the distribution package.
+. In this case the code NOJCN means 'Contract Notice (CN) published on a National Government Official Journal' (see file /codelists/gc/docref-content-type.gc in the distribution package).
```
Comment (arillpa) [inline modules/ROOT/pages/2.6_EU_and_national_publications.adoc:201]: see file /codelists/gc/docref-content-type.gc, I think we need a link here

File: modules/ROOT/attachments/dist/xslt/XLSX CodeLists to Genericode/README.txt [hunk 15]
```diff
@@ -3,9 +3,9 @@ README
 
 Steps to generate the GC files out from the ESPD-EDM Code Lists excel file:
 
-1. Open the file "https://github.com/ESPD/ESPD-EDM/blob/3.0.0/docs/src/main/asciidoc/dist/cl/xlsx/ESPD-CodeLists-V3.0.0.xlsx"
-2. Save it locally as "ESPD-CodeLists-V3.0.0.xml" (Select the following filetype: XML Spreadsheet)
-3. Use your preferred XLS-T transformation method and the Stylesheet "Excel_to_GC_Multilang_v1.0.xsl" as an input to generate the *.gc files (each tab in "ESPD-CodeLists-V02.00.01.xlsx" will be transformed into a separate *.gc file)
+1. Open the file "https://github.com/ESPD/ESPD-EDM/tree/3.0.1/ESPD-EDM/__ESPDTeam__/ESPD-CodeLists.xlsx"
+2. Save it locally as "ESPD-CodeLists.xml" (Select the following filetype: XML Spreadsheet)
+3. Use your preferred XLS-T transformation method and the Stylesheet "excel-to-gc-multilang-v1.0.xsl" as an input to generate the *.gc files (each tab in "ESPD-CodeLists.xlsx" will be transformed into a separate *.gc file)
```
Comment (arillpa) [inline modules/ROOT/attachments/dist/xslt/XLSX CodeLists to Genericode/README.txt:8]: Comment for the commit below - image [modules/ROOT/images/Distribution_package.jpg](https://github.com/OP-TED/espd-docs/pull/1/commits/1cb651802963332db7fcd5b0620871499f33bc4f#diff-9d6a815335d012509ee2ab516ec166e72f53de0ece1b8251a4272115184872d3)

Te added image has 'docs' between 'data-model' and java-library', is that correct?

File: modules/ROOT/pages/2.4_Root_elements.adoc [hunk 16]
```diff
@@ -127,6 +127,11 @@ In ESPD v3.0.0 the contracting party information will come from eForms. Which al
 
 *Rule*: (see examples further below in this document)
 
+|*cac:TenderingCriterion/cac:ProcurementProjectLotReference* |Associated class |0..n |One or more of the procurement project lots to which this criterion can be related to. a|
```
Comment (arillpa) [inline modules/ROOT/pages/2.4_Root_elements.adoc]: this comment refers to commit above for the image. The picture in the documentation Figure 12 does not correspond to this diagram, for example 'TenderingCriterionXXXX**cbcID** in the documentaion but not in the diagram. 
I think the documentation needs update

File: modules/ROOT/pages/3.6_Data_Structures.adoc [hunk 17]
```diff
@@ -18,7 +18,7 @@ The columns of the tables are to be interpreted as follows:
...
 * Column 23: *ElementUUID*, The Universal Unique Identifier assigned to identify unambiguously a criterion, group or subgroup. These UUID are kept in e-Certis, except for those that have to be generated dynamically (e.g. UUIDs for Questions, Captions and Requirements). See also the special note "Criteria Taxonomy/Data Structures and the use of UUIDS" below
-* Column 24: *Element Code*; in the case of Criterion it contains the code that categorises the criterion in the taxonomy of exclusion and selection criteria (it maps to the taxonomy drawn in the online documentation and in the spread-sheet files "ESPD-CriteriaTaxonomy*" in folder dist/cl). For the rest of elements there are three types of codes (see also section:
+* Column 24: *Element Code*; in the case of Criterion it contains the code that categorises the criterion in the taxonomy of exclusion and selection criteria (it maps to the taxonomy drawn in the online documentation and in the spread-sheet files "ESPD-criterion*" in folder /codelists/). For the rest of elements there are three types of codes (see also section:
```
Comment (arillpa) [inline modules/ROOT/pages/3.6_Data_Structures.adoc]: The sentence seems incomplete
'For the rest of elements there are three types of codes (see also section:'

File: modules/ROOT/pages/3.7_XML_examples_and_tools.adoc [hunk 18]
```diff
@@ -3,7 +3,7 @@ include::partial$attributes.adoc[]
 
 The fact of presenting the data structures as a spread-sheet book had an additional reason: *to use the spread-sheet as an elementary prototype tool to generate the XML instances of the criteria for the ESPD Request and ESDP Response documents*.
 
-Thus the folder link:{url-tree}/dist/xslt/[dist/xslt/ODS Data Structures to ESPD XML] contains four XSL style-sheets that facilitate the generation of the complete set of criteria required in an ESDP Request or in an ESDP Response XML file.
+Thus the folder link:{url-tree}/xml-examples/__xslt__/ods-data-structures-to-ESPD-xml[xml-examples/__xslt__/ods-data-structures-to-ESPD-xml] contains four XSL style-sheets that facilitate the generation of the complete set of criteria required in an ESDP Request or in an ESDP Response XML file.
```
Comment (arillpa) [inline modules/ROOT/pages/3.7_XML_examples_and_tools.adoc]: link:{url-tree}/xml-examples/__xslt__/ods-data-structures-to-ESPD-xml[xml-examples/__xslt__/ods-data-structures-to-ESPD-xml] 

link does not work.

File: modules/ROOT/pages/Tech_AnnexII_XML_Examples.adoc [hunk 19]
```diff
@@ -1,9 +1,10 @@
-== Annex II. XML Examples
+= Annex II. XML Examples
+include::partial$attributes.adoc[]
 
-=== AII.1. Code List Examples
+== AII.1. Code List Examples
 
 [[access-right-xml-example,access-right xml Example]]
-==== AII.1.. access-right
```
Comment (arillpa) [inline modules/ROOT/pages/Tech_AnnexII_XML_Examples.adoc]: The numbering A II.1.. for all code list examples is a bit confusing but we can live with it for the time being.
Is it a reason, why it has to be like this?

File: modules/ROOT/nav.adoc [hunk 20]
```diff
@@ -0,0 +1,11 @@
+* <<xml_business_handbook.adoc#, Business Handbook>>
+* Technical Handbook
```
Comment (arillpa) [inline modules/ROOT/nav.adoc:2]: when the navigation was added in November we realized that there was a problem with the with of the columns in the text of the TechHandbook.
I see now has been corrected now. 
However all xml examples seem now to be cut on the right side. Is that a consequence of resizing the columns in the tables?

**Review by rousso — CHANGES_REQUESTED** [review]
Thank you  Pedro. Please do the changes that were suggested in Paloma's comments and resubmit the PR.

File: modules/ROOT/pages/6.5_Reference_to_publications_and_to_ESPD_Request.adoc (hunk 6 above)
Comment (psotofer) [reply modules/ROOT/pages/6.5_Reference_to_publications_and_to_ESPD_Request.adoc:69]: Fixed.

File: modules/ROOT/pages/5.5_Financial_ratios.adoc (hunk 7 above)
Comment (psotofer) [reply modules/ROOT/pages/5.5_Financial_ratios.adoc:39]: Scaped the underscore chars. Fixed.

File: modules/ROOT/pages/6.5_Reference_to_publications_and_to_ESPD_Request.adoc (hunk 8 above)
Comment (psotofer) [reply modules/ROOT/pages/6.5_Reference_to_publications_and_to_ESPD_Request.adoc]: Fixed.

File: modules/ROOT/pages/6.6_Economic_Operator.adoc (hunk 9 above)
Comment (psotofer) [reply modules/ROOT/pages/6.6_Economic_Operator.adoc]: Updated link text as 'Country'.

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (psotofer) [reply modules/ROOT/pages/index.adoc]: Added link to the conceptual model.

File: modules/ROOT/pages/1.5_Codes_and_Identifiers.adoc (hunk 11 above)
Comment (psotofer) [reply modules/ROOT/pages/1.5_Codes_and_Identifiers.adoc:149]: Fixed.

File: modules/ROOT/pages/3.4_Properties.adoc (hunk 12 above)
Comment (psotofer) [reply modules/ROOT/pages/3.4_Properties.adoc:59]: Updated link text.

File: modules/ROOT/pages/3.7_XML_examples_and_tools.adoc (hunk 13 above)
Comment (psotofer) [reply modules/ROOT/pages/3.7_XML_examples_and_tools.adoc:6]: Fixed for another comment.

File: modules/ROOT/pages/3.7_XML_examples_and_tools.adoc [hunk 21]
```diff
@@ -19,7 +19,7 @@ The following features *are implemented* in the first set of transformation XSL-
 
 The following features *are NOT implemented* in the first set of transformation XSL-T style-sheet (ESPDRequest-Annotated-V3.0.0.xslt):
 
-* The publications and document references requested in the business requirements are not generated; but the XML examples provided in the distribution do contain examples of TED and national publications (for the ESPDRequest.xml and ESPDResponse.xml, in the link:{url-tree}/dist/xml-examples[xml-examples folder].
+* The publications and document references requested in the business requirements are not generated; but the XML examples provided in the distribution do contain examples of TED and national publications (for the ESPDRequest.xml and ESPDResponse.xml, in the link:{url-tree-tmp}/dist/xml-examples[xml-examples folder].
```
[the remark this answers is not in the document]
Comment (psotofer) [reply modules/ROOT/pages/3.7_XML_examples_and_tools.adoc:22]: Fixed link.

File: modules/ROOT/pages/3.7_XML_examples_and_tools.adoc [hunk 22]
```diff
@@ -31,5 +31,5 @@ The following features *are implemented* in the second set of transformation XSL
 
 The following feature *is NOT implemented* in the first set of transformation XSL-T style-sheet (ESPDRequest-Annotated-V3.0.0.xslt):
 
-* The Criteria from the ESPD Request *are not* copied in the ESPD Response document. But the XML examples in the link:{url-tree}/dist/xml-examples/[xml-examples folder].
+* The Criteria from the ESPD Request *are not* copied in the ESPD Response document. But the XML examples in the link:{url-tree-tmp}/dist/xml-examples/[xml-examples folder].
```
[the remark this answers is not in the document]
Comment (psotofer) [reply modules/ROOT/pages/3.7_XML_examples_and_tools.adoc:34]: Fixed link.

File: modules/ROOT/pages/2.6_EU_and_national_publications.adoc (hunk 14 above)
Comment (psotofer) [reply modules/ROOT/pages/2.6_EU_and_national_publications.adoc:201]: Added link to code list.

File: modules/ROOT/pages/2.6_EU_and_national_publications.adoc [hunk 23]
```diff
@@ -193,10 +193,10 @@ The XML snippet below illustrates how to use the cac:AdditionalDocumentReference
 ----
 
 . For TED Notices you must use the Publications Office Official Journal Supplement number in its original format [][][][]/S [][][]-[][][][][][], e.g. 2017/S 142-293520. Notice that the schemeAgencyID is EU-COM-OP (OP standing for 'Publications Office').
-. The jurisdiction scope of this document is typified in the Code List docref-content-type (see file dist/cl/ESPD-CodeLists-V3.0.0 in the distribution package). *Beware that the name and content of this Code List has changed from the previous ESPD-EDM version 2.1.1*.
+. The jurisdiction scope of this document is typified in the Code List docref-content-type (see file /codelists/gc/docref-content-type.gc in the distribution package). *Beware that the name and content of this Code List has changed from the previous ESPD-EDM version 2.1.1*.
```
[the remark this answers is not in the document]
Comment (psotofer) [reply modules/ROOT/pages/2.6_EU_and_national_publications.adoc:196]: Added link.

File: modules/ROOT/attachments/dist/xslt/XLSX CodeLists to Genericode/README.txt (hunk 15 above)
Comment (psotofer) [reply modules/ROOT/attachments/dist/xslt/XLSX CodeLists to Genericode/README.txt:8]: Fixed image removing docs folder.

File: modules/ROOT/pages/2.4_Root_elements.adoc (hunk 16 above)
Comment (psotofer) [reply modules/ROOT/pages/2.4_Root_elements.adoc]: Diagram should be fixed now.

File: modules/ROOT/pages/3.7_XML_examples_and_tools.adoc [hunk 24]
```diff
@@ -22,7 +22,7 @@ The following features *are NOT implemented* in the first set of transformation
 * The publications and document references requested in the business requirements are not generated; but the XML examples provided in the distribution do contain examples of TED and national publications (for the ESPDRequest.xml and ESPDResponse.xml, in the link:{url-tree}/xml-examples[xml-examples folder].
 * The response value and cardinality are shown for informative purposes. No functionality is currently implemented based on them, but could be used in future improved versions of the prototype;
 
-The following features *are implemented* in the second set of transformation XSL-T style-sheet (From-REQUEST_to_RESPONSE.xslt files):
+The following features *are implemented* in the second set of transformation XSL-T style-sheet (From-REQUEST-to-RESPONSE.xslt files):
```
[the remark this answers is not in the document]
Comment (psotofer) [reply modules/ROOT/pages/3.7_XML_examples_and_tools.adoc:25]: Added link to the file.

File: modules/ROOT/pages/3.6_Data_Structures.adoc (hunk 17 above)
Comment (psotofer) [reply modules/ROOT/pages/3.6_Data_Structures.adoc]: Removed the incomplete sentence.

File: modules/ROOT/pages/3.7_XML_examples_and_tools.adoc (hunk 18 above)
Comment (psotofer) [reply modules/ROOT/pages/3.7_XML_examples_and_tools.adoc]: Fixed.

File: modules/ROOT/pages/Tech_AnnexII_XML_Examples.adoc (hunk 19 above)
Comment (psotofer) [reply modules/ROOT/pages/Tech_AnnexII_XML_Examples.adoc]: The intention was to give an specific numbering for the Annex; in the end it was never finished.

File: modules/ROOT/nav.adoc (hunk 20 above)
Comment (psotofer) [reply modules/ROOT/nav.adoc:2]: Due to the length of the xml code I'm afrait it cannot be helped. Nonetheless, each xml snippet has a scroll bar at the bottom to view each line in full.
Maybe we could split xml snippets in smaller blocs so navigation is easier.
Captura

File: modules/ROOT/pages/Tech_AnnexII_XML_Examples.adoc (hunk 19 above)
Comment (psotofer) [reply modules/ROOT/pages/Tech_AnnexII_XML_Examples.adoc]: Numbering fixed.

**Pull request merged**